app/assets/stylesheets/application.css.scss

```diff
@@ -44,7 +57,26 @@ body[data-action="devise/sessions-new"], body[data-action="devise/sessions-creat
 }
 
 #footer {
-  font-size: 10px;
+  background-color: $navs-bg-color;
+  bottom: 0;
+  height: 20px;
+  left: 0;
+  line-height: 20px;
+  position: absolute;
+  vertical-align: middle;
+  width: 100%;
+
+  p {
+    font-size: 10px;
+    margin: 0;
+    text-align: center;
+
+  a {
```

Comment:
Line should be indented 4 spaces, but was indented 2 spaces<br>Selector should have depth of applicability no greater than 2, but was 3